.github/actions/setup-and-build/action.yml

```diff
@@ -0,0 +1,25 @@
+name: Setup and build
```

Comment:
This is a local Github action that can be used to reduce repetition. See https://github.com/stackblitz/tutorialkit/pull/106/commits/cddfde9deeb9ca01d7b05c894eceaba16c97e354 for the affected files.